### ISSUE TITLE
zlib 1.2.8 CVE Security Issue

### ISSUE THREAD

**Issue by BrettThePark** [barteksc/AndroidPdfViewer#788]
Version 2.8.2 relies on ZLIB 1.2.8 (via pdfium) which has a number of CVE's issued. 

https://github.com/barteksc/PdfiumAndroid/issues/66

**Comment by liu564166123** [comment]
Yes, it has the following CVE problems, wish a stable version which can solve it.
![image](https://user-images.githubusercontent.com/72922867/126981280-5a7257b9-f79b-464b-9792-bb9033a1ffaf.png)
![image](https://user-images.githubusercontent.com/72922867/126981290-d1a01dab-6e06-46f9-b26a-238fadd4e094.png)
![image](https://user-images.githubusercontent.com/72922867/126981297-c387eefa-68d9-4835-b5ff-65cc2bc89dda.png)
![image](https://user-images.githubusercontent.com/72922867/126981307-94a2ec62-b35c-4cc3-b2d7-4eccbea892f7.png)
